e_ductus/tests/test_e_ductus.py

```diff
@@ -26,7 +27,47 @@ def test_resolve_to_home_page_view(self):
 class BaseViewTest(TestCase):
     def test_resolve_to_base_page_view(self):
         resolver = resolve('/')
-        self.assertEqual(resolver.func.view_class, BaseView)
+        self.assertEqual(resolver.func.view_class,BaseView)
+
+
+class CourseDeleteViewTest(TestCase):
+    def test_resolve_to_course_delete_page_view(self):
+        resolver = resolve('/<pk>/delete/')
+        self.assertEqual(resolver.func.view_class,CourseDeleteView)
```

Comment:
There is no point testing resolver, resolver is tested well enough by Django. And there is very little room for errors in `urls.py` file. It is much more useful to test actual requests with Django test client, which will also test the routing.